x-pack/solutions/observability/plugins/infra/public/components/asset_details/components/section_title.tsx

```diff
@@ -28,19 +28,22 @@ export const TitleWithTooltip = ({
   'data-test-subj': dataTestSubject,
   tooltipTestSubj,
   tooltipContent,
+  tooltipAriaLabel,
 }: {
   title: string;
   tooltipContent: ReactNode;
   'data-test-subj'?: string;
   tooltipTestSubj?: string;
+  tooltipAriaLabel?: string;
 }) => {
+  const ariaLabel = tooltipAriaLabel ?? title;
   return (
     <EuiFlexGroup gutterSize="xs" alignItems="center" responsive={false}>
       <EuiFlexItem grow={false}>
         <SectionTitle title={title} data-test-subj={dataTestSubject} />
       </EuiFlexItem>
       <EuiFlexItem grow={false}>
-        <Popover icon="iInCircle" data-test-subj={tooltipTestSubj}>
+        <Popover icon="iInCircle" data-test-subj={tooltipTestSubj} aria-label={ariaLabel}>
```

Comment:
You could set it directly here, but it's up to you.

```suggestion
        <Popover icon="iInCircle" data-test-subj={tooltipTestSubj} aria-label={tooltipAriaLabel ?? title}>
```